src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  --color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+.nav_link {
+  display: block;
+  position: relative;
+  color: #000;
+  text-decoration: none;
+  height: 60px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav-item:not(:last-child) {
+  padding-right: 20px;
+}
+
+.logo {
+  position: absolute;
```

Comment:
Why do you need position absolute here?

Reply:
without it, the top doesn't work. But the margin worked, removed the absolute. I made an analogy with the search bar...

Reply:
In cases when using  paddings/margins is enough better don't use absolute